packages/eas-cli/src/commands/build/run.ts

```diff
@@ -132,7 +139,7 @@ async function resolvePlatformAsync(platform?: string): Promise<AppPlatform> {
     choices: [
       { title: 'Android', value: AppPlatform.Android },
       { title: 'iOS', value: AppPlatform.Ios },
-    ],
+    ].filter(choice => choice.value !== AppPlatform.Ios || process.platform === 'darwin'),
```

Comment:
Do we want to prompt non-Mac users to select a platform having only one option available? Or just always select Android for them if the platform flag is not specified?

I suppose we want to just select Android but what is your opinion?

Reply:
let's select android by default